src/sentry/static/sentry/app/data/timezones.tsx

```diff
@@ -293,6 +292,7 @@ export default [
   ['Africa/Djibouti', '(UTC+0300) Africa/Djibouti'],
   ['Africa/Juba', '(UTC+0300) Africa/Juba'],
   ['Africa/Kampala', '(UTC+0300) Africa/Kampala'],
+  ['Europe/Kiev', '(UTC+0300) Europe/Kiev'],
```

Comment:
nit: Can we group this will the other Europe/UTC+0300 entries?

Reply:
@billyvg totally agree with that. I grouped it using K in Kiev ignoring the region for some reason. Updated.